packages/synthetix-main/contracts/interfaces/IAccountModule.sol

```diff
@@ -0,0 +1,16 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@synthetixio/core-contracts/contracts/interfaces/ISatelliteFactory.sol";
+
+interface IAccountModule is ISatelliteFactory {
+    function initializeAccountModule() external;
+
+    function isAccountModuleInitialized() external view returns (bool);
+
+    function upgradeAccountImplementation(address newAccountImplementation) external;
+
+    function getAccountAddress() external view returns (address);
+
+    function getAccountModuleSatellites() external view returns (Satellite[] memory);
```

Comment:
im assuming its returning an array only because that is how general satellite implementations work? Normally the synthetix system should only need a single satellite for purposes of the account token, but I may not understand how satellites work in this case or what the purpose should be here

Reply:
Exactly, it is an array for all kind of satellites so that the CLI let's you interact with this kind of Modules (only one satellite) or with multiple satellites